_pages/ato/types.md

```diff
@@ -3,15 +3,16 @@ title: Types of ATO
 navtitle: Types
 ---
 
-There are three tracks of documentation available:
+There are four tracks of authorization available:
 
+* [Pre-authorization for internal government use](#pre-authorization)
 * [90-day Lightweight Authority to Operate (LATO)](#day-authorization)
 * [1-year Lightweight Authority to Operate (LATO)](#year-authorization)
 * [Continuous Authority to Operate (cATO)](#continuous-authorization)
 
 ### Pre-authorization
 
-You may operate without authorization if all of the following conditions are met:
+You may operate without further authorization, based on our approved pre-existing security authorization, if all of the following conditions are met:
```

Comment:
I don't see how it implies - seems like it says it directly. In the same way, I don't see pre-authorization as implying anything about its duration, but happy to take suggestions on re-naming. I don't think I'm the one who even named it, and if I was, it was offhand in a meeting.